File: Logic_HandsOn/Java/Problems on Array,Pattern etc/Program87.java
/////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
//	Program:
//		Accept Character from user and check whether it is capital or not (A-Z).
//
/////////////////////////////////////////////////////////////////////////////////////////////////////////////////

import java.util.*;

class Solution
{

	public char ch;

	public Solution(char ch)
	{
		this.ch = ch;
	}

	public boolean ChkCapital()
	{	
		
		if(ch>='A' && ch<='Z')
		{
			return true;
		}
		else
		{
			return false;
		}
	}


}

class Program
{
	public static void main(String[] args)
	{
		Scanner sObj = new Scanner(System.in);
		boolean bRet = false;

		System.out.println("Enter character:");
		char ch = sObj.next().charAt(0);

		Solution obj = new Solution(ch);
		bRet = obj.ChkCapital();

		if(bRet == true)
		{
			System.out.println("It is Capital");
		}
		else
		{
			System.out.println("It is not Capital");
		}
	}
}

/*
OUTPUT:

Enter character:
F
It is Capital

*/

File: Logic_HandsOn/C++/Problems on Digits,String,Array,Pointers etc/program87.cpp
/////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
//	Program:
//		Accept Character from user and check whether it is capital or not (A-Z).
//
/////////////////////////////////////////////////////////////////////////////////////////////////////////////////

#include<iostream>
using namespace std;

class Solution
{
	private:
		char ch;
		
	public:
		Solution(char ch='\0')
		{
			this->ch = ch;
		}

		bool ChkCapital()
		{
			if(ch>='A' && ch<='Z')
			{
				return true;
			}
			else
			{
				return false;
			}


		}


};

int main()
{
	char ch='\0';
	bool bRet=false;

	cout<<"Enter the character: ";
	cin>>ch;

	Solution *dObj = new Solution(ch);
	bRet = dObj->ChkCapital();

	if(bRet==true)
	{
		cout<<"It is Capital\n";
	}
	else
	{
		cout<<"It is not Capital\n";
	}

	delete dObj;
	return 0;
}

/*
OUTPUT:
Enter the character: F
It is Capital

*/
